lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -300,6 +300,8 @@ sub scenario_name {
 
 sub scenario_description {
     my ($self) = @_;
+    my $description = $self->settings_hash->{TEST_SUITE_DESCRIPTION};
```

Comment:
I would go one step further and always get the description from the settings.

Reply:
I don't understand

Reply:
If `TEST_SUITE_DESCRIPTION` is not defined the code continues to check the `TestSuites` database( via `$self->scenario`). I'd suggest we never want to look at that table. We might not have a test suite at all. See also #2610 for the brainstorming branch which also makes that change.

Arguably you can leave this to a follow-up, but I wanted to mention it here, since this would be the long-term goal.

Reply:
I've just read this after my review and it is basically the same as I suggested in the other comment. So I would say that makes conceptually sense. My only worry is the increased database size due to all the copying. We needed to write a migration to implement this anyways so when doing that migration would see how the sizes increases.